### PR TITLE
fix: build under 1.8

### LINKED ISSUE #259: mvn clean package: error, Could not find artifact javax.jnlp:jnlp:jar:1.6 at specified path /usr/lib/jvm/java-11-openjdk/lib/javaws.jar
```
Tests run: 191, Failures: 0, Errors: 0, Skipped: 0

[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ delta-common ---
[INFO] Building jar: /home/guoyi/Downloads/open-delta/common/target/delta-common-1.0.3-SNAPSHOT.jar
[INFO] 
[INFO] ------------------< au.org.ala.delta:delta-ui-common >------------------
[INFO] Building DELTA User Interface Common 1.0.3-SNAPSHOT                [2/9]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Open DELTA suite 1.0.3-SNAPSHOT:
[INFO] 
[INFO] DELTA Common ....................................... SUCCESS [  6.292 s]
[INFO] DELTA User Interface Common ........................ FAILURE [  0.012 s]
[INFO] DIST ............................................... SKIPPED
[INFO] KEY ................................................ SKIPPED
[INFO] intkey ............................................. SKIPPED
[INFO] DELTA Editor ....................................... SKIPPED
[INFO] CONFOR ............................................. SKIPPED
[INFO] DELFOR ............................................. SKIPPED
[INFO] Open DELTA suite ................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.509 s
[INFO] Finished at: 2022-12-02T10:38:01+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project delta-ui-common: Could not resolve dependencies for project au.org.ala.delta:delta-ui-common:jar:1.0.3-SNAPSHOT: Could not find artifact javax.jnlp:jnlp:jar:1.6 at specified path /usr/lib/jvm/java-11-openjdk/lib/javaws.jar -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :delta-ui-common
```

### PR DESCRIPTION
**Pull request by starsareintherose**
close #259

@chrisala

### CODE REVIEW

File: ui-common/pom.xml [hunk 1]
```diff
@@ -157,6 +157,13 @@
 			<version>2.0.05</version>
 			<type>jar</type>
 		</dependency>
+		<dependency>
+			<groupId>javax.jnlp</groupId>
+			<artifactId>jnlp</artifactId>
+			<version>1.6</version>
+			<scope>system</scope>
+			<systemPath>/usr/share/icedtea-web/javaws.jar</systemPath>
```
Comment (chrisala) [inline ui-common/pom.xml:165]: This systemPath probably should be a property ${local.jnlp.path} rather than hard coded to /usr/share/icedtea-web

**Review by chrisala — COMMENTED** [review]
Thanks for the PR @starsareintherose 
Would you mind making the path to the local jnlp.jar installation a property?

File: ui-common/pom.xml (hunk 1 above)
Comment (starsareintherose) [reply ui-common/pom.xml:165]: okay, that's fine, I will fix it

**Comment by starsareintherose** [comment]
That's fine, I will handle it soon

**Review by chrisala — APPROVED** [review]
Thanks @starsareintherose

**Pull request merged**